code/game/gamemodes/cult/runes.dm

```diff
@@ -711,7 +711,7 @@
 			T.imbue = "emp"
 			imbued_from = R
 			break
-		if(R.word1==cultwords["blood"] && R.word2==cultwords["see"] && R.word3==cultwords["destroy"]) //conceal
+		if(R.word1==cultwords["hide"] && R.word2==cultwords["see"] && R.word3==cultwords["blood"]) //conceal
```

Comment:
It's not something I expect you to fix, but this basically only happened because some cleverclogs decided that the talisman code should exactly copy rune code and then made a mistake, instead of putting it all in one place.